src/java.base/share/classes/java/lang/ThreadLocal.java

```diff
@@ -120,13 +120,15 @@ private static int nextHashCode() {
      * most once per thread, but it may be invoked again in case of
      * subsequent invocations of {@link #remove} followed by {@link #get}.
      *
-     * <p>This implementation simply returns {@code null}; if the
+     * @implSpec
+     * This implementation simply returns {@code null}; if the
      * programmer desires thread-local variables to have an initial
-     * value other than {@code null}, {@code ThreadLocal} must be
-     * subclassed, and this method overridden.  Typically, an
-     * anonymous inner class will be used.
+     * value other than {@code null}, then either {@code ThreadLocal} can be
+     * subclassed and this method overridden or {@link ThreadLocal#withInitial(Supplier)} method
+     * can be used to construct a {@code ThreadLocal}.
```

Comment:
This looks okay but maybe slightly better to say "or the method withInitial(Supplier) can be used ...". 

I think you can probably reflow this paragraph to avoid having a mix of short and long lines.

Reply:
Hello Alan, I've updated this PR with this suggested change and also formatted this new text to avoid having a mix of long and short lines.